--- a/.vite/build/workspace-root-drop-handler-CAzIvt6a.js
+++ b/.vite/build/workspace-root-drop-handler-CAzIvt6a.js
@@ -40225,6 +40225,7 @@
     i = t.Ei(e.version);
   return (
     Fz({
+      enabled: !1,
       dsn: t.Ti,
       environment: tB.buildFlavor,
       release: i,
--- a/.vite/build/worker.js
+++ b/.vite/build/worker.js
@@ -74306,6 +74306,7 @@
   let n = NH(e.appVersion),
     r = e.buildFlavor !== `prod`;
   (vO({
+    enabled: !1,
     dsn: wW,
     environment: e.buildFlavor,
     release: TW(n.version),
