--- a/webview/assets/app-main-Cykt_nvm.js
+++ b/webview/assets/app-main-Cykt_nvm.js
@@ -25234,6 +25234,6 @@
 function dP({ appVersion: e, client: t, deviceId: n, enabled: r }) {
   let { data: i, status: a } = Oc(),
-    o = r && a === `success` && i === !0,
+    o = false,
     s = (0, $.useMemo)(() => fP(t), [t]),
     c = (0, $.useRef)(o),
     l = (0, $.useRef)(null),
@@ -25373,6 +25373,7 @@
   };
 }
 async function hP(e) {
+  return;
   try {
     switch (e.eventKind) {
       case `turn_rating`:
@@ -25920,10 +25920,13 @@
   XP = 500,
   ZP = `1569253508`,
   QP = {
+    overrideAdapter: window.__ELECTRON_SHIM__.overrideAdapter,
+    disableLogging: true,
     networkConfig: {
       api: KP,
       logEventUrl: tP,
       sdkExceptionUrl: qP,
+      preventAllNetworkTraffic: true,
       networkOverrideFunc: UP,
     },
   };
@@ -26004,6 +26007,7 @@
   );
 }
 function eF(e) {
+  return e.children;
   let t = (0, Q.c)(27),
     {
       auth: n,
--- a/webview/assets/composer-DdM3sB3u.js
+++ b/webview/assets/composer-DdM3sB3u.js
@@ -3127,6 +3127,7 @@
   isReferralCta: r,
   referralCtaText: i,
 }) {
+  return false;
   let a = { ...t },
     o = {
       eventType: e,
