--- a/webview/assets/chunk-LFPYN7LY-h_ROgi7V.js
+++ b/webview/assets/chunk-LFPYN7LY-h_ROgi7V.js
@@ -1061,7 +1061,14 @@
   unstable_useTransitions: a,
 }) {
   let s = r.useRef();
-  s.current ??= o({ initialEntries: n, initialIndex: i, v5Compat: !0 });
+  s.current ??= o({
+    initialEntries: n ?? [
+      window.__ELECTRON_SHIM__.initialRoute ??
+        window.location.pathname + window.location.search + window.location.hash,
+    ],
+    initialIndex: i,
+    v5Compat: !0,
+  });
   let c = s.current,
     [l, u] = r.useState({ action: c.action, location: c.location }),
     d = r.useCallback(
--- a/webview/assets/app-shell-state-CGqmfzBu.js
+++ b/webview/assets/app-shell-state-CGqmfzBu.js
@@ -562,7 +562,7 @@
 }
 var Et = `app-shell-bottom-panel-launcher-visible`,
   Dt = 100,
-  G = i(a, !0),
+  G = i(a, window.__ELECTRON_SHIM__.initialSidebarState),
   K = i(a, !1),
   q = i(a, !1),
   J = i(a, !1),
@@ -588,7 +588,7 @@
       };
     },
   }),
-  At = i(a, () => new d(1)),
+  At = i(a, () => new d(window.__ELECTRON_SHIM__.initialSidebarState ? 1 : 0)),
   jt = i(a, wt),
   X = ie(Et, !0),
   Mt = i(k, !1),
